flow/raster_cache.h

```diff
@@ -129,10 +125,9 @@ class RasterCache {
   const size_t access_threshold_;
   const size_t picture_cache_limit_per_frame_;
   size_t picture_cached_this_frame_ = 0;
-  PictureRasterCacheKey::Map<Entry> picture_cache_;
-  LayerRasterCacheKey::Map<Entry> layer_cache_;
+  mutable PictureRasterCacheKey::Map<Entry> picture_cache_;
+  mutable LayerRasterCacheKey::Map<Entry> layer_cache_;
```

Comment:
Is this actually necessary? Where is the non-const access happening in a const context?